.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ permissions:
   contents: read 
 
 env: 
-  MSRV: 1.69.0
+  MSRV: 1.71.0
```

Comment:
We should not change the MSRV for an architecture, I think we should add a step that will overwrite the `MSRV` environment variable to 1.71.0 when `${{ matrix.target }}` is `loongarch64-unknown-linux-gnu`, something like:

```yaml
- name: change Rust toolchain version for Loongarch
  run: echo "MSRV=$(if [ '${{ matrix.target }}' = 'loongarch64-unknown-linux-gnu' ]; then echo '1.71.0'; else echo '${{ env.MSRV }}'; fi)" >> $GITHUB_ENV
```